accord-core/src/test/java/accord/impl/basic/Cluster.java

```diff
@@ -475,11 +513,40 @@ public static Map<MessageType, Stats> run(Id[] nodes, MessageListener messageLis
             }, 5L, SECONDS);
 
             Scheduled reconfigure = sinks.recurring(configRandomizer::maybeUpdateTopology, 1, SECONDS);
+
+            Scheduled purge = sinks.recurring(() -> {
+                trace.debug("Triggering purge.");
```

Comment:
It would be nice to randomise the frequency of this (lots of time in tests is now spent on this), and also to randomise the set of nodes we shutdown for this, so that some transactions can carry on being processed (potentially) during this window, and we can exercise more interesting situations

Reply:
Added randomization.